ci/docker/pact-broker/Dockerfile

```diff
@@ -0,0 +1 @@
+FROM pactfoundation/pact-broker:2.54.0.0
```

Comment:
Some EOF characters have sneaked into this file and the one above - adding a new line to each file should clean that up.